Update package-lock.json and deck.gl build

ilan-gold opened the pull request:
I figured I would knock out updating the `deck.gl` dependencies while I was updating for https://github.com/babel/babel/issues/11216.  This begins to fix https://github.com/hubmapconsortium/vitessce/issues/436

manzt commented on CHANGELOG.md:
```diff
@@ -2,6 +2,12 @@
 
 ## 0.1.0 - In Progress
```
We've released 0.1.0 right? I submitted a PR this morning and made 0.1.1 in progress

ilan-gold replied:
Yes looks like I messed up the deployment.  The scripts should work but I think I didn't update the CHANGELOG in anticipation of the new scripts properly.

manzt replied:
Cool. I'll mirror this in my PR.

ilan-gold commented:
I updated the `version.sh` so now here is a before and after, which is I think more what we want:
Before:
```
# Changelog

## In Progress

### Changed

- Update deck.gl build
- Bump `zarr.js` to `v0.2.3`
- Set raster selection on data loader class

### Added

- Add `test:watch` to npm scripts.

## 0.1.0

### Changed

- Clean up shaders
- Extracted data utils to loaders and refactored demo

### Added

- StaticImageLayer added and exported for viewing non-tiled images
- Single channel colormaps for tiled and static
- Expose opacity and visibility
- Testing for layers added
- More automated build process
```

After:
```
# Changelog

## In Progress

### Added

### Changed

## 0.1.1

### Changed

- Update deck.gl build
- Bump `zarr.js` to `v0.2.3`
- Set raster selection on data loader class

### Added

- Add `test:watch` to npm scripts.

## 0.1.0

### Changed

- Clean up shaders
- Extracted data utils to loaders and refactored demo

### Added

- StaticImageLayer added and exported for viewing non-tiled images
- Single channel colormaps for tiled and static
- Expose opacity and visibility
- Testing for layers added
- More automated build process
```
